src/support/examples/triangle/shader/triangle_110.glslv

```diff
@@ -1,8 +1,8 @@
-#version 150 core
+#version 110
```

Comment:
I didn't mean to make any of these changes, sorry for the confusion and extra code to review!